### PR TITLE
Adding disabled feature to report button

### PR DESCRIPTION
**Pull request by OscarDeDios**
Adding disabled atribute to the report button.
https://github.com/systelab/allure-reporter/issues/64

### CODE REVIEW

File: src/app/features/reporter/reporter-dialog.component.html [hunk 1]
```diff
@@ -66,5 +66,5 @@
 </div>
 <systelab-dialog-bottom>
 	<button type="button" class="btn mr-0" (click)="doUpdateTestCase()">Update Test Case</button>
-	<button type="button" class="btn ml-auto" (click)="doRun()">Report</button>
+	<button type="button" class="btn ml-auto" [disabled]="!isValidForm()" (click)="doRun()">Report</button>
```
Comment (olgapuig) [inline src/app/features/reporter/reporter-dialog.component.html:69]: The conditions to disable this button must be Project, Test Plan, Test Cycle, Test Group and New Test Cycle name. All of them must be met. I suggest to change isValidForm to include all of them.

File: src/app/features/reporter/reporter-dialog.component.html [hunk 2]
```diff
@@ -66,5 +66,5 @@
 </div>
 <systelab-dialog-bottom>
 	<button type="button" class="btn mr-0" (click)="doUpdateTestCase()">Update Test Case</button>
```
Comment (olgapuig) [inline src/app/features/reporter/reporter-dialog.component.html:68]: The button Update Test Case must be disabled when the project field is not informed. Please, add the [disabled]=!this.selectedProjectId

**Review by olgapuig — CHANGES_REQUESTED** [review]

File: src/app/features/reporter/reporter-dialog.component.html (hunk 1 above)
Comment (OscarDeDios) [reply src/app/features/reporter/reporter-dialog.component.html:69]: I think the method is already correct, what could we add more?
```
	public isValidForm() {
		if (this._userId) {
			if (this.selectedTestCycleId) {
				return true;
			} else {
				return this.selectedProjectId && this.selectedTestPlanId && this.selectedTestGroups.length > 0 && this.nameForNewTestCycle !== '';
			}
		} else {
			return false;
		}
	}
```

Comment (olgapuig) [reply src/app/features/reporter/reporter-dialog.component.html:69]: Both buttons must be disabled on this case 
image

The condition if (this.selectedTestCycleId) { return true; } must be removed and include all the fields in just one

**Review by olgapuig — APPROVED** [review]

**Review by atzurk — APPROVED** [review]

**Pull request merged**